Sources/ConstraintConstantTarget.swift

```diff
@@ -27,49 +27,88 @@
     import AppKit
 #endif
 
-
 public protocol ConstraintConstantTarget {
+    func asFloat() -> CGFloat?
+    func asSize() -> CGSize?
+    func asPoint() -> CGPoint?
+    func asConstraintInsets() -> ConstraintInsets?
+
+#if canImport(UIKit)
+    func asConstraintDirectionalInsets() -> ConstraintDirectionalInsets?
+#endif
 }
 
 extension CGPoint: ConstraintConstantTarget {
+    public func asFloat() -> CGFloat? {
+        nil
+    }
+    public func asSize() -> CGSize? {
+        return nil
+    }
+    public func asPoint() -> CGPoint? {
+        return self
+    }
+    public func asConstraintInsets() -> ConstraintInsets? {
+        return nil
+    }
+#if canImport(UIKit)
+    public func asConstraintDirectionalInsets() -> ConstraintDirectionalInsets? {
+        return nil
+    }
+#endif
 }
 
-extension CGSize: ConstraintConstantTarget {    
+extension CGSize: ConstraintConstantTarget {
+    public func asFloat() -> CGFloat? {
+        nil
+    }
+    public func asSize() -> CGSize? {
+        return self
+    }
+    public func asPoint() -> CGPoint? {
+        return nil
+    }
+    public func asConstraintInsets() -> ConstraintInsets? {
+        return nil
+    }
+#if canImport(UIKit)
+    public func asConstraintDirectionalInsets() -> ConstraintDirectionalInsets? {
+        return nil
+    }
+#endif
 }
 
-extension ConstraintInsets: ConstraintConstantTarget {
-}
+extension ConstraintInsets: ConstraintConstantTarget {}
 
 #if canImport(UIKit)
 @available(iOS 11.0, tvOS 11.0, *)
 extension ConstraintDirectionalInsets: ConstraintConstantTarget {
+    public func asFloat() -> CGFloat? {
+        nil
+    }
+    public func asSize() -> CGSize? {
+        return nil
+    }
+    public func asPoint() -> CGPoint? {
+        return nil
+    }
+    public func asConstraintInsets() -> ConstraintInsets? {
+        return nil
+    }
+    public func asConstraintDirectionalInsets() -> ConstraintDirectionalInsets? {
+        return self
+    }
```

Comment:
No need of explicit return statement. Follow the way you coded in line 43 asFloat() function